src/actions/wrap.py

```diff
@@ -1,14 +1,63 @@
+from typing import Union
 from ..paired_delimiter import paired_delimiters_map
-from talon import Module
+from talon import Module, actions, app
+from ..csv_overrides import init_csv_and_watch_changes
 
 
 mod = Module()
 
 
 mod.list("cursorless_wrap_action", desc="Cursorless wrap action")
+mod.list("cursorless_wrapper_snippet", desc="Cursorless wrapper snippet")
 
 
-@mod.capture(rule=("{user.cursorless_paired_delimiter}"))
-def cursorless_wrapper(m) -> list[str]:
-    paired_delimiter_info = paired_delimiters_map[m.cursorless_paired_delimiter]
-    return [paired_delimiter_info.left, paired_delimiter_info.right]
+# NOTE: Please do not change these dicts.  Use the CSVs for customization.
+# See https://github.com/pokey/cursorless-talon/blob/main/docs/customization.md
+wrapper_snippets = {
+    "else state": "cursorless.wrapper.ifElseStatementElseBranch",
+    "if else": "cursorless.wrapper.ifElseStatementIfBranch",
+    "if state": "cursorless.wrapper.ifStatement",
```

Comment:
Note: this term is configured independently from "if state" the scope type. Could be surprising to users but I wasn't sure of the right approach here because there are multiple ways to wrap something with an if statement and I could see people wanting to wrap differently from how they refer to scope type maybe?  Also, fwiw, I have configured this to "if" for myself, because I think I can get away with shorter name than when it's used for a scope type